modules/ballerina-builtin/src/main/ballerina/ballerina/security/natives.bal

```diff
@@ -0,0 +1,40 @@
+package ballerina.security;
+
+@Description { value:"Returns a hash of a given string using the SHA-256 algorithm "}
+@Param { value:"baseString: The string to be hashed" }
+@Param { value:"algorithm: The hashing algorithm to be used" }
+@Return { value:"The hashed string" }
+public native function getHash (string baseString, string algorithm) (string);
```

Comment:
Is this security related function?

Reply:
This is used to generate SHA1, SHA256, MD5 hashes which are used for security purposes. But we can move them to a separate package(crypro package) as well.